### PR TITLE
Physics-informed neural network experiments

### PR DESCRIPTION
**Pull request by djgagne**
I have added a notebook `notebooks/physics_based_mp_nn.ipynb` that contains my rough experiments with a mass conserving neural net. Now I want to work with Gabrielle to modify the main code pipelines to incorporate the new additions.

### CODE REVIEW

**Comment by djgagne** [comment]
The CI is now passing. @ggantos can you review and approve? After that you can merge the changes to the github actions file with your branch, and then your CI should also pass.

**Review by ggantos — APPROVED** [review]

**Pull request merged**
